Dockerfile

```diff
@@ -145,7 +155,7 @@ RUN ${BUILD_DIR}/src/guacd-docker/bin/build-all.sh
 RUN ${BUILD_DIR}/src/guacd-docker/bin/list-dependencies.sh \
         ${PREFIX_DIR}/sbin/guacd               \
         ${PREFIX_DIR}/lib/libguac-client-*.so  \
-        ${PREFIX_DIR}/lib/freerdp2/*guac*.so   \
+        ${PREFIX_DIR}/lib/freerdp3/*guac*.so   \
```

Comment:
What happens, here, if someone specifies that they want to build `WITH_FREERDP=2.x`?

Reply:
Updated so now you can use 2 or 3 such as 

`docker build --build-arg FREERDP_VERSION=2 -t guacamole-server-rdp .`